core-web/apps/dotcms-ui/src/app/view/components/login/dot-login-component/dot-login.component.ts

```diff
@@ -116,11 +116,15 @@ export class DotLoginComponent implements OnInit, OnDestroy {
         this.setInitialMessage(loginInfo);
     }
 
+    private isEmail(potentialEmail: string): boolean {
+        return !!new FormControl(potentialEmail, Validators.email).errors?.email;
```

Comment:
@victoralfaro-dotcms I suggest we could avoid creating a FormControl to validate the email instead we could use a regex to check, unfortunately the [email regex](https://github.com/angular/angular/blob/80f472f9f4c09af33f41f7e8dd656eff0b74d03f/packages/forms/src/validators.ts#L139) is not exported by Angular but we could copy the regex and use it for example:

```ts
const EMAIL_REGEXP =
  /^(?=.{1,254}$)(?=.{1,64}@)[a-zA-Z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-zA-Z0-9!#$%&'*+/=?^_`{|}~-]+)*@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/;

private isEmail(potentialEmail: string): boolean {
        return EMAIL_REGEXP.test(potentialEmail)
  }
```

We could even create this function as a utility to check an email without needing to instantiate a FormControl.

Reply:
Yes.